### ISSUE TITLE
Scrollbar Same color as page background

### ISSUE THREAD

**Issue by timkingbu** [bu-ist/responsive-framework#299]
**Describe the bug**
The scrollbar color is the same as the page background, making it hard to see or interact with.

**To Reproduce**
1. Go to https://id-demos.cms-devl.bu.edu/bu-profiles/
2. Scroll anywhere
3. The scrollbar is white and appears over the dark header background, but can't be seen on the white page background.

**Expected behavior**
Scrollbar should be visible. On this page, the browser sees the page background as dark (defined in _normalize-mixin.scss by body { background-color: #000; }) so it makes the scrollbar white. Not sure if this is an edge case or if it appears elsewhere.

**Device Information:**
https://www.whatsmybrowser.org/b/3T3MK
